README.md

```diff
@@ -15,17 +15,21 @@ AzureHPC Node Health Checks provides an automated suite of test that targets spe
 - [NDm H100 v5-series](https://learn.microsoft.com/en-us/azure/virtual-machines/nd-h100-v5-series)
 - [NDm A100 v4-series](https://learn.microsoft.com/en-us/azure/virtual-machines/ndm-a100-v4-series)
 - [ND A100 v4-series](https://learn.microsoft.com/en-us/azure/virtual-machines/nda100-v4-series)
+- [NC A100 v4-series](https://learn.microsoft.com/en-us/azure/virtual-machines/nc-a100-v4-series)
 - [HBv4-series](https://learn.microsoft.com/en-us/azure/virtual-machines/hbv4-series)
 - [HX-series](https://learn.microsoft.com/en-us/azure/virtual-machines/hx-series)
 - [HBv3-series](https://learn.microsoft.com/en-us/azure/virtual-machines/hbv3-series)
 - [HBv2-series](https://learn.microsoft.com/en-us/azure/virtual-machines/hbv2-series)
 - [NCv3-series](https://learn.microsoft.com/en-us/azure/virtual-machines/ncv3-series)
 
+
```

Comment:
Should there be one or two spaces here?

Reply:
I don't think this is a big deal since Markdown doesn't really care. It was added for my own readability but I can remove as it probably just adds confusion.

Reply:
addressed